### PR TITLE
Optimization of nm journey API

### PR DESCRIPTION
**Pull request by djludo**

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by antoine-de** [comment]
jenkins, test this please

File: source/routing/raptor.h [hunk 1]
```diff
@@ -129,7 +129,7 @@ struct RAPTOR
                    const uint32_t max_transfers,
                    const type::AccessibiliteParams & accessibilite_params,
                    const std::vector<std::string> & forbidden_uri,
-                   bool clockwise);
+                   bool clockwise, bool details);
```
Comment (TeXitoi) [inline source/routing/raptor.h]: as each parameters has its own line, it'll be more coherent that details has its own line.

**Comment by TeXitoi** [comment]
with the small comment, OK for me.

**Comment by antoine-de** [comment]
jenkins, test this please

**Pull request merged**